src/controllers/userController.ts

```diff
@@ -0,0 +1,85 @@
+import { Response, Request, NextFunction } from 'express';
+import { User } from '../models/User';
+
```

Comment:
For more type safety, we should make a type `type UserType = 'donor' | 'volunteer' | 'recipient' | 'admin' | 'any'` and use `userType: UserType` instead of `userType: string`